### ISSUE TITLE
MS account belongs to someone under 18

### ISSUE THREAD

**Issue by Vortecked** [Kqzz/MCsniperGO#42]
microsoft account belongs to someone under 18! add to gamily for this to work

getting this error, i changed the birth date to something over 18, added to a family (where my other account is the owner)

any fix?

**Comment by 6uf** [comment]
does it work now? if not then wait a while ig. it prob takes a bit for it to update on microsofts end /shrug 

if it doesnt work after a bit just try to change it again wait. no idea bro

**Comment by Vortecked** [comment]
yeah sorry i kept trying and after a couple more days and it finally worked
